### ISSUE TITLE
FETCH NEXT CURSOR - unused_variables - false positive

### ISSUE THREAD

**Issue by Zeatynis** [abaplint/abaplint#3930]
Below code gives false positive on `unused_variables` for `package_size `variable.
```
REPORT zfoo.

CONSTANTS package_size TYPE i VALUE 5000.

DATA documents TYPE STANDARD TABLE OF bkpf WITH KEY bukrs belnr gjahr.
DATA db_cursor TYPE cursor.

START-OF-SELECTION.

  OPEN CURSOR WITH HOLD db_cursor FOR
      SELECT * FROM bkpf.

  DO.
    FETCH NEXT CURSOR cursor
        INTO TABLE documents
        PACKAGE SIZE package_size.
  ENDDO.
```

/zfoo.prog.abap[3, 11]: Variable "package_size" not used(unused_variables)

**Comment by larshp** [comment]
fixed

<img width="1390" height="894" alt="Image" src="https://github.com/user-attachments/assets/0a4cfe05-0559-4a6a-b8b2-60814937e14b" />
